services/admin/backend/api/handler/project.go

```diff
@@ -60,43 +59,42 @@ func createProject(service project.UseCase) http.Handler {
 		// convert input strings to value objects
 		sc, err := valueobject.NewShortCode(input.ShortCode)
 		if err != nil {
-			w.WriteHeader(http.StatusInternalServerError)
+			w.WriteHeader(http.StatusBadRequest)
```

Comment:
What's the benefit of change `StatusInternalServerError` to `StatusBadRequest`?

Reply:
That's logic. I've already approved this PR, just wanted to know the background of the change, which seems to be right.

Reply:
@mdelez I agree.